src/main/java/com/jalasoft/selenium/ronald/movies/Customer.java

```diff
@@ -0,0 +1,73 @@
+package com.jalasoft.selenium.ronald.movies;
+
+import java.util.Enumeration;
+import java.util.Vector;
+
+/**
+ * Customer class.
+ * @author ronald_butron
+ */
+class Customer {
+    private String name;
+    private Vector rentals = new Vector();
+
+    /**
+     * Constructor.
+     *
+     * @param name customer name.
+     */
+    Customer(final String name) {
+        this.name = name;
+    }
+
+    /**
+     * Add rental.
+     * @param arg Rental objected
+     */
+    public void addRental(final Rental arg) {
+        rentals.addElement(arg);
+    }
+
+    /**
+     * Get costumer name.
+     * @return customer name
+     */
+    public String getName() {
+        return name;
+    }
+
+    /**
+     * Calculate statement value fo account.
+     *
+     * @return statement of account.
+     */
+    public String statement() {

```

Comment:
you can split this method
currently this method is not following the Single responsibility principle
It is doing 3 things, it is calculating totalAmount, totalFrequentRenterPoints and it is printing too.